ext/opentelemetry-ext-grpc/tests/test_server_interceptor.py

```diff
@@ -49,6 +53,36 @@ def service(self, handler_call_details):
 
 
 class TestOpenTelemetryServerInterceptor(TestBase):
+    def test_instrumentor(self):
+        def handler(request, context):
+            return b""
+
+        grpc_server_instrumentor = GrpcInstrumentorServer()
+        grpc_server_instrumentor.instrument()
+
+        server = grpc_server_instrumentor.server
```

Comment:
This seems like the wrong flow, it should look more like
```
GrpcInstrumentorServer().instrument()
server = grpc.server(...)
```
It seems like you are using `instrument()` to replace the `grpc.server()` call, instead I think you should be _wrapping_ `server()` so when the user calls `grpc.server()` the wrapper will then call `intercept_server()`